op-challenger/game/fault/types/types_test.go

```diff
@@ -32,17 +34,17 @@ func TestIsRootPosition(t *testing.T) {
 	}{
 		{
 			name:     "ZeroRoot",
-			position: NewPositionFromGIndex(0),
+			position: NewPositionFromGIndex(common.Big0),
```

Comment:
```suggestion
			position: NewPositionFromGIndex(big.NewInt(0)),
```

For good measure I guess. :)